src/tests/cgpath.c

```diff
@@ -104,33 +104,33 @@ static int test_running_container(const char *lxcpath, const char *name)
 		goto err3;
 
 	/* test get/set value using memory.soft_limit_in_bytes file */
-	ret = cgroup_ops->get(cgroup_ops, "memory.soft_limit_in_bytes", value,
-			      sizeof(value), c->name, c->config_path);
-	if (ret < 0) {
+	ret = cgroup_ops->get(cgroup_ops, "pids.max", value, sizeof(value),
```

Comment:
Won't those changes break on non-cgroup2?

Reply:
No, non-cgroup2 has `pids.max` too and the Jenkins runners run on non-cgroup2. :)

Reply:
doh, read cpu.max :)